addons/mail/views/mail_templates.xml

```diff
@@ -274,7 +274,7 @@
                 <script type="text/javascript" src="/mail/static/src/messaging/component_hook/use_store/use_store_tests.js"></script>
                 <script type="text/javascript" src="/mail/static/src/messaging/entity/attachment/attachment_tests.js"></script>
                 <script type="text/javascript" src="/mail/static/src/messaging/entity/message/message_tests.js"></script>
-                <script type="text/javascript" src="/mail/static/src/messaging/entity/partner/partner_tests.js"></script>
+                <script type="text/javascript" src="/mail/static/src/messaging/entity/partner/messaging_tests.js"></script>
```

Comment:
As it is, this should be moved into messaging folder? Also why not in initializer after all?

Reply:
messaging initializer is an implementation detail, especially when current partner relation is set on messaging. That's why it makes more sense for this being a messaging test rather than messaging initializer test.